docs/documentation/server_admin/topics/threat/oauth21-compliance.adoc

```diff
@@ -2,4 +2,4 @@
 === OAuth 2.1 compliance
 
 To make sure that {project_name} server will validate your client to be more secure and OAuth 2.1 compliant, you can configure client policies
-for the OAuth 2.1 support. Details are described in the OAuth 2.1 section of link:{adapterguide_link}#_oauth21-support[{adapterguide_name}].
+for the OAuth 2.1 support. *OAuth 2.1* details are described in the link:https://www.keycloak.org/guides#securing-apps[securing apps] section.
```

Comment:
Same like my other comment (can be hopefully attribute instead of hardcoded link).